src/ansys/fluent/core/utils/file_transfer_service.py

```diff
@@ -154,7 +154,7 @@ def __init__(
         )
         self.image_tag = image_tag if image_tag else "latest"
         self.mount_target = mount_target if mount_target else "/home/container/workdir/"
-        self.mount_source = mount_source if mount_source else pyfluent.USER_DATA_PATH
+        self.mount_source = mount_source if mount_source else pyfluent.EXAMPLES_PATH
```

Comment:
Just thinking out loud here on seeing this: it seems it would be good if we could avoid mentioning `pyfluent.EXAMPLES_PATH` inside constructors (for example). Any kind of injection would be preferable IMHO.

Reply:
@seanpearsonuk @mkundu1 @prmukherj 

The injected path should not be `pyfluent.EXAMPLES_PATH` otherwise it will give the file modification issue while using file transfer service.